src/tasks/tap/link.js

```diff
@@ -66,17 +66,25 @@ const buildTapObj = async (globalConfig, silent, name, location) => {
  * @returns {void}
  */
 const linkTap = async args => {
-  const { command, globalConfig, options, params, tasks } = args
+  const { globalConfig, params } = args
+
   const { name, location, silent } = params
 
+  // get the tap config located at our current location to determine the default alias name
+  const [ tapConfig ] = getTapConfig({ path: location })
+
+  // get the tap name, or throw an error if not specified
+  const alias = name || tapConfig.alias || generalError(
```

Comment:
While `tapConfig.alias` works, I would prefer that its name-spaced inside the `keg` property object
* The tap config get's merged with the `app.json` from keg-core
* At some point, well need to setup expo-js to load this merged file
  * Currently it's using the root `app.json` file of keg-core so it's not an issue
* Once that's done, we'll want all `keg` related items name-spaced inside the `keg` key
  *  In anticipation of this, `keg-herkin` has the `alias` already in the `tap.js` file at `keg.cli.link.name`
  * `keg.cli.link.name` might be more than it needs. `keg.alias` would work as well, as it's still under the `keg` property
* Doing the above means, it would be accessed with `tapConfig.keg.alias` instead of `tapConfig.alias`

Reply:
updated